packages/model/src/research-output.ts

```diff
@@ -221,9 +221,13 @@ export type ResearchOutputCreateDataObject = ResearchOutputCoreObject & {
   organismIds: string[];
   subtypeId?: string;
   teamIds: string[];
+  addedDate: string;
 };
 
-export type ResearchOutputUpdateDataObject = ResearchOutputCoreObject & {
+export type ResearchOutputUpdateDataObject = Omit<
+  ResearchOutputCoreObject,
+  'addedDate'
```

Comment:
ok so this is what i was getting at in some comments above - i think the update should allow the modification of addedDate